src/AnalyticsEventService.ts

```diff
@@ -0,0 +1,23 @@
+import {AnalyticsConfig} from "./AnalyticsConfig";
+import {EventAPIResponse, EventPayload} from "./models";
```

Comment:
I think you might need to rebase.

Reply:
Hmm rebasing adds all my previous commits to the list but I think I have to in order for the checks to pass

Reply:
I think I'm just gonna start over with a new PR...